packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -212,8 +212,8 @@ export function getPropGetters<
           // could reopen the panel once they resolve.
```

Comment:
on line above this: s/running/pending/

Reply:
Good catch, I found two occurrences that I fixed [here](de4a205c).